Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContent.js

```diff
@@ -1,2 +1,10 @@
 import React from 'react';
 import './Header.css';
+
+const HeaderContent = () => (
+    <div>
+        <p>Let's learn React by building simple interfaces with components. Don't try to overthink it, just keep it simple and have fun. Once you feel comfortable using components you are well on your way to mastering React!</p>
+    </div>
```

Comment:
Does the content need to be wrapped in a div and a p tag? Could you get away with just a p?